src/commands/publish.ts

```diff
@@ -31,36 +31,50 @@ import {
     validateCategory,
     validateInputs,
 } from "../utils";
-
-const publish = async (
-    name: string | null,
-    category: string,
-    categories?: string[]
-): Promise<void> => {
+import { integrationIdFromName } from "../utils/validation";
+
+const publish = async ({
+    name,
+    category,
+    categories,
+    integrationName,
+}: {
+    name: string | null;
+    category: string;
+    categories?: string[];
+    integrationName?: string;
+}): Promise<void> => {
```

Comment:
**Suggestion:** naming this interface could make it a little more readable. Great to have types here either way, though!

Reply:
I think in this  case, having the types and function parameters close together is probably the most readable option.